.golangci.yaml

```diff
@@ -12,3 +12,12 @@ linters:
     - stylecheck
     - unconvert
     - unused 
+linters-settings:
+  stylecheck:
+    # ST1005: Incorrectly formatted error string
+    checks: ["all", "-ST1005"]
+  gosec:
+    excludes:
+      # TODO: Identify, fix, and remove violations of most of these rules
```

Comment:
nit: file a bug for this and reference the bug number

Reply:
lmao I'm pretty sure that one in Boulder is my fault, too 🙃 I've grown as a person since then!